apps/files/src/components/FileFilterMenu.vue

```diff
@@ -1,26 +1,25 @@
 <template>
-  <v-menu class="mt-2" offset-y>
-    <v-btn slot="activator" flat @click="focusFilenameFilter"><v-icon large>filter_list</v-icon></v-btn>
-    <v-list>
-      <v-list-tile v-for="(filter, fid) in fileFilter" :key="fid">
-        <v-list-tile-title v-text="fileFilterTranslations[filter.name]"></v-list-tile-title>
-        <v-checkbox :input-value="filter.value" @change="setFileFilter({ name: filter.name, value: $event })"></v-checkbox>
-      </v-list-tile>
-      <v-list-tile>
-        <v-list-tile-title>
-          <span v-translate>Name Filter</span>
-        </v-list-tile-title>
-        <v-list-tile-title>
-        <search-bar @input="setFilterTerm" :value="filterTerm" ref="filenameFilter" autofocus :label="$gettext('Filter')" icon="false"/>
-        </v-list-tile-title>
-      </v-list-tile>
-    </v-list>
-  </v-menu>
+  <ul class="uk-nav uk-dropdown-nav uk-nav-default">
```

Comment:
use oc-drop, oc-nav and oc-nav-item as per https://github.com/owncloud/owncloud-design-system/pull/226 ?

Reply:
Oc-drop is now there but there is a little problem now with tests because of missing oc-menu in other components. I'll have a look and create a new PR to fix it.